rn/Teacher/src/modules/dashboard/Dashboard.js

```diff
@@ -437,7 +452,8 @@ export class Dashboard extends React.Component<Props, State> {
           accessibilityLabel: i18n('Edit Dashboard'),
           action: this.showFavoritesList,
           disabled: !this.props.totalCourseCount || Boolean(this.props.pending),
-        }],
+        },
+        ...devOnlyButtons],
```

Comment:
Is all this just to avoid dev tapping into somewhere? Would a temporary scheme launch argument be better?

Reply:
I was getting tired of having to tap either through the app or into the developer menu every time I wanted to check a change so I added this button for development only to take me to the last screen I was on. I can remove if others don't want this but this was a life saver for me from having to go into speed grader every time.

Reply:
I personally dislike launch arguments because I always forget to remove them.

Reply:
I can be ok with this if it makes your life easier. ;)